source/Octopus.Tentacle.Contracts/CapabilitiesResponse.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+
+namespace Octopus.Tentacle.Contracts
+{
+    public class CapabilitiesResponse
```

Comment:
This is strategically an envelope so that we can extend the response later if required.